arcus_orientation.md

```diff
@@ -694,9 +694,32 @@ The [training videos](#training-videos) walk through everything you need to get
 
 ***
 
-## Lab Security
+## Security Feature: Internet Disabled Mode
 
-(here's where general content about why we disable internet goes)
+While there are myriad benefits to taking a scripted approach to data analysis, it does also introduce some specific privacy and security risks. Thankfully, Arcus has taken these risks into consideration, and developed our [Internet Disabled Mode](https://forum.arcus.chop.edu/t/about-the-arcus-lab-security-setting-for-disabling-internet-in-labs/741) feature. The feature provides peace of mind by allowing you to work on your code in an environment that is entirely disconnected from the internet, keeping your data and code safer. 
+
+There are two primary kinds of problems that could occur: 
+
+1. **Malicious Packages**
+
+    Writing code involves downloading a variety of packages to use in your scripts, which could contain malicious code (code which is intended to harm a system or cause a security breach). [Malicious packages can be found even popular package repositories like PyPI](https://thehackernews.com/2023/11/27-malicious-pypi-packages-with.html), and in some instances, [well-known, legitimate packages have been hacked and had malicious code inserted](https://therecord.media/malware-found-in-npm-package-with-millions-of-weekly-downloads).
+
+2. **Internet Reliant Features**
+
+    Some legitimate, non-malicious packages are useful in part because they send data out to external servers where additional processing is done. Unfortunately, it is not always easy to tell when a package relies upon sending data over the internet. This may not be a concern for a hobby programmer, but when working with PHI and other sensitive data, this presents a huge risk. 
+
+Using Internet Disabled Mode helps protect you from both of these risks, by allowing you to work on your code in a space that is entirely disconnected from the internet. 
+
+But how do I install packages without internet?
+------
+Of course, you'll still need to be able to install packages to do your work. For this, we've developed the **Package Installation Form**. Rather than installing your packages in the same environment that includes your data, filling out this form instantly installs the necessary packages in a separate environment that doesn't have any data in it. This ensures that, even if there is any malicious code in the package you've installed, it doesn't have any access to your data when it executes. The package is then immediately made available for your use in your Internet Disabled lab. 
```

Comment:
* "instantly" may not be the exact right wording here. The intent is to to convey that no approval process is required, but the package installation can be expected to take a bit longer than using conventional installation approaches.
* You probably want to mention the download capability of the install form. This additional use case (in addition to brevity) is why we internally gravitated to the moniker "Install Form".
* Re: malicious code, there are two aspects. 1) During installation itself, if the package _installation scripts_ (as opposed to the package itself) are malicious, then there is no access to data. 2) When the package is actually used in the lab (e.g. run as part of an R or Python analysis program), if the package is malicious, the package will have access to the data, but it will at least not be able to "phone home" to a malicious server to exfiltrate data or download additional malicious software. I know it's hard to convey the nuance here.

Reply:
Thanks for these comments, Kevin! I'll try to come up with some adjusted wording -- to better convey the nuance without being too wordy!